lib/prompts/expand.js

```diff
@@ -218,13 +218,18 @@ Prompt.prototype.validateChoices = function (choices) {
 /**
  * Generate a string out of the choices keys
  * @param  {Array}  choices
- * @param  {Number} defaultIndex - the choice index to capitalize
+ * @param  {Number|String} default - the choice index or name to capitalize
  * @return {String} The rendered choices key string
  */
-Prompt.prototype.generateChoicesString = function (choices, defaultIndex) {
+Prompt.prototype.generateChoicesString = function (choices, defaultChoice) {
   var defIndex = choices.realLength - 1;
-  if (_.isNumber(defaultIndex) && this.opt.choices.getChoice(defaultIndex)) {
-    defIndex = defaultIndex;
+  if (_.isNumber(defaultChoice) && this.opt.choices.getChoice(defaultChoice)) {
+    defIndex = defaultChoice;
+  } else if (_.isString(defaultChoice)) {
+    let index = _.flatMap(choices.realChoices, function(c) {
```

Comment:
Missed that earlier in the PR, but we can do all this in a single loop if we use Lodash `_.findIndex`

`_.findIndex(choices.realChoices, ({value}) => value === defaultChoice)`

(and let's do the same with the rawlist to keep the logic more consistent)

Reply:
Sure, that sounds great.